Sources/Nimble/Matchers/Predicate.swift

```diff
@@ -312,7 +312,7 @@ final public class NMBPredicateStatus: NSObject {
     public static let doesNotMatch: NMBPredicateStatus = NMBPredicateStatus(status: 1)
     public static let fail: NMBPredicateStatus = NMBPredicateStatus(status: 2)
 
-    public override var hashValue: Int { return self.status.hashValue }
+    public override var hash: Int { return self.status.hashValue }
```

Comment:
Is this compatible with Swift 4.1?

Reply:
I'll check the warning in Swift 4.2 later.

Reply:
https://github.com/apple/swift-corelibs-foundation/blob/master/Foundation/NSObject.swift#L326

Reply:
> `NSObject` implements this by returning `self.hash`. Subclasses can
    customize hashing by overriding the `hash` property.

Reply:
https://github.com/apple/swift/blob/6fc4cef671947d1d2482951853f7f82854473d91/test/Inputs/clang-importer-sdk/swift-modules-without-ns/ObjectiveC.swift#L77

Reply:
Totally makes sense :+1: Thanks for the input!